doc/source/whatsnew/v0.18.1.txt

```diff
@@ -428,6 +428,30 @@ In addition to this error change, several others have been made as well:
 - ``pd.read_csv()`` no longer allows a combination of strings and integers for the ``usecols`` parameter (:issue:`12678`)
 
 
+.. _whatsnew_0181.api.to_datetime:
+
+``to_datetime`` error changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Bugs in ``pd.to_datetime()`` when passing a ``unit`` with convertible entries and ``errors='coerce'`` or non-convertible with ``errors='ignore'`` (:issue:`11758`)
+
+Previous behaviour:
+
+.. code-block:: python
+
+   In [27]: pd.to_datetime(1420043460, unit='s', errors='coerce')
+   Out[27]: NaT
+
+   In [28]: pd.to_datetime(11111111, unit='D', errors='ignore')
+   OverflowError: Python int too large to convert to C long
+
+New behaviour:
+
+.. ipython:: python
+
+   pd.to_datetime(1420043460, unit='s', errors='coerce')
+   pd.to_datetime(11111111, unit='D', errors='ignore')
```

Comment:
You show the changes for `errors='ignore'` and `errors='coerce'`, but I think the default behaviour (for `errors='raise'`) also changed? (from OverflowError to ValueError)? 
Maybe good to add this case as well to the examples